PROTOCOL.md

```diff
@@ -596,6 +612,51 @@ The following is an example `domainMetadata` action:
 }
 ```
 
+### Sidecar File Information
```

Comment:
We should probably update the Action Reconciliation section to specifically state that v2 checkpoint actions are not allowed in normal commit files, and do not participate in log replay?